### PR TITLE
fix(styles): make survey iframe responsive

### PR DESCRIPTION
**Pull request by johngruen**

### CODE REVIEW

File: packages/fxa-payments-server/server/config/index.js [hunk 1]
```diff
@@ -243,7 +243,7 @@ const conf = convict({
     },
     surveyGizmo: {
       url: {
-        default: 'https://www.surveygizmo.com',
+        default: 'https://qsurvey.mozilla.com/s3/post-subscribe-survey',
```
[the remark this answers is not in the document]
Comment (johngruen) [reply packages/fxa-payments-server/server/config/index.js]: ah, yeah, i changed the survey URL...and forgot to update the way the url gets read in that file...IDK how different environments work with the config-based URL though.

[the remark this answers is not in the document]
Comment (lmorchard) [reply packages/fxa-payments-server/server/config/index.js]: @johngruen probably need to consult with Marty about changing that URL... I don't have access to the survey stuff, but

1. it needs to be an iframe embed - not sure if this new URL is
1. there's specific JS code configured on the Survey Gizmo side that uses `postMessage` to signal to us that the survey has been completed

[the remark this answers is not in the document]
Comment (lmorchard) [reply packages/fxa-payments-server/server/config/index.js]: Also, that config setting is used for CSP headers to allow iframe embed. IIRC, it needs to be a base URL and not a full survey URL.

[the remark this answers is not in the document]
Comment (lmorchard) [reply packages/fxa-payments-server/server/config/index.js]: I would say we should move changing this URL to a different PR... that's a bit more than just making the iframe responsive :)

[the remark this answers is not in the document]
Comment (johngruen) [reply packages/fxa-payments-server/server/config/index.js]: okay i will revert the stuff with the new URL...we should change that later...will file an issue separately.

**Pull request merged**